modules/msgpack/src/main.cpp

```diff
@@ -0,0 +1,126 @@
+#include <msgpack_module.hpp>
+
+MAKE_ENTRY_POINT()
+
+namespace ArkMsgpack
+{
+    Value pack(const std::vector<Value> &args)
+    {
+        // pack(Value src) and return a buffer's string packed 
+        if(args.size() != 1)
+            throw std::runtime_error("ArgError : This function must have 1 arguments");
+        ValueType type {args[0].valueType()};
+        std::stringstream buffer;
+        CObjekt value_src {get_cobjekt(args[0], type)};
+        Value packed;
+        
+        buffer.seekg(0);
+        if(type == ValueType::NFT)
+        {
+            auto src {std::get<bool>(value_src)};
```

Comment:
Avoid using of msgpack when you can

Reply:
I don't understand...

Reply:
Oops, autocorrect wrote msgpack instead of `auto`.

I mean that using `auto` is not a practice I recommend because reading the type (as a developper) is important to understand the code. Using `auto` is fine to me when we have an obvious type which is quite long, for example an iterator.

Reply:
Yes, I can see